manifest-c9s.yaml

```diff
@@ -25,7 +25,8 @@ repos:
   - baseos
   - appstream
   - sig-nfv
-  # Temporarily include RHCOS 9 repo for cri-o, cri-tools, oc & hyperkube
+  - okd-copr
+  # Temporarily include RHCOS 9 repo for oc & hyperkube
```

Comment:
Can you update the `includepkg` list in `ci/prow-entrypoint.sh` too?

Reply:
updated over in https://github.com/openshift/os/pull/993